src/main/java/org/ow2/proactive/catalog/repository/entity/CatalogObjectEntity.java

```diff
@@ -108,7 +108,7 @@ public int hashCode() {
     private CatalogObjectEntityKey id;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @MapsId("bucketId")
+    @MapsId("bucketName")
```

Comment:
Since we choose to keep using the composite key, this MapsId value should not be changed, revert it back to bucketId